Monika After Story/game/zz_music_selector.rpy

```diff
@@ -32,6 +32,8 @@ init -1 python in songs:
     SAYO_NARA_SENS = "Sayonara"
     PLAYWITHME_VAR6 = "Play With Me (Variant 6)"
     YR_EUROBEAT = "Your Reality (Eurobeat ver.)"
+    MONIKA_LULLABY = "Monika's Lullaby"
+    THIRTY_MIN_OF_SILENCE = "30 Minutes of Silence"
```

Comment:
the 30 minutes doesnt have to have a filename.

I think it might be worth making the lullably as an unlockable song, but we can do that in a separate pull req